docs/highlighters/regexp.md

```diff
@@ -12,11 +12,32 @@ To use this highlighter, associate regular expressions with styles in the
 
 ```zsh
 typeset -A ZSH_HIGHLIGHT_REGEXP
-ZSH_HIGHLIGHT_REGEXP+=('\bsudo\b' fg=123,bold)
+ZSH_HIGHLIGHT_REGEXP+=('^rm .*' fg="red",bold)
 ```
 
-This will highlight "sudo" only as a complete word, i.e., "sudo cmd", but not
-"sudoedit"
+This will highlight the whole line starting with `rm` command (for all
+operating systems, in contrast to the below example).
+
+Some regular expressions are [subject to the host platform][MAN_ZSH_REGEX], 
+especially the kernel. To highlight `sudo` only as a complete word, i.e., 
+`sudo cmd`, but not `sudoedit`, the respective regular expressions for the host
+systems would be:
+
+* GNU-Linux
```

Comment:
I believe this is usually stylized as `GNU/Linux`.

Reply:
Good point.  It _is_ usually stylized as you say, but let's be more accurate and state "GNU libc (e.g., most Linux distros)", as per your other comment.  The other bullet can remain as-is because on BSD platforms libc isn't as independent/swappable a component as it is on Linux.

Reply:
> Good point. It _is_ usually stylized as you say, but let's be more accurate and state "GNU libc (e.g., most Linux distros)", as per your other comment. The other bullet can remain as-is because on BSD platforms libc isn't as independent/swappable a component as it is on Linux.

I will also take this part! Thank you both.